pwiz_tools/Skyline/TestFunctional/FullScanGraphTest.cs

```diff
@@ -298,6 +298,39 @@ private void TestSpecialIonsAnnotations()
             //check that the special ion annotation shows in both library and full scan viewers
             TestAnnotations(new [] {"Reporter_Test+"});
             TestLibraryMatchAnnotations(new[] { "Reporter_Test+" });
+            Settings.Default.ShowSpecialIons = false;
+        }
+
+        private void TestIonMatchToleranceUnitSetting()

```

Comment:
To test this more thoroughly will require using the TransitionSettingsUI form and also a case of using PPM in the Import Peptide Search wizard to exercise the TransitionSettingsControl.

Reply:
I replaced the changing the settings  with invocation of the Settings dialogue and added tolerance unit verification to the ImportPeptideSearchTest.